.idea/lessoner_project.iml

```diff
@@ -183,4 +183,9 @@
       </list>
     </option>
   </component>
+  <component name="RakeTasksCache">
```

Comment:
Remove this file from the commit. By some reason this file is still here.

Reply:
Also resolve merge conflicts.

Reply:
<img width="856" alt="image" src="https://user-images.githubusercontent.com/7360925/186087355-49674c2e-dc30-41f0-82ea-5398e06133bf.png">